### DISCUSSION TITLE
Job Title: PHP Laravel Full Stack Developer (2-5 years experience) Location: Noida

### DISCUSSION THREAD

**Discussion by mukund101985** [community/community#147897] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Job Title: PHP Laravel Full Stack Developer (2-5 years experience)
Location: Noida (Work from Office)
Key Requirements:
* 2-5 years of experience in PHP and Laravel
* Proficiency in front-end technologies (HTML, CSS, JavaScript)
* Experience with databases (MySQL)
* Strong understanding of RESTful APIs and version control (Git)
* Must be able to work from office in Noida
* Salary upto 8 LPA

DM your resume with current CTC & notice period

**Comment by mecodeatlas** [comment]
Hi @mukund101985, we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
